ts/src/gate.ts

```diff
@@ -284,21 +284,6 @@ export default class gate extends Exchange {
                             'sub_accounts/{user_id}/keys/{key}': 2.5,
                         },
                     },
-                    'portfolio': {
-                        'get': {
```

Comment:
@sc0Vu there endpoints were added recently, are you sure they were 100% deprecated already?

![image](https://github.com/ccxt/ccxt/assets/43336371/ae67e94b-8814-40d7-8b36-5ec3284ab7a8)

Reply:
@sc0Vu the confirmed that portfolio APIs are still working, don't remove them here